### ISSUE TITLE
Broken link in README

### ISSUE THREAD

**Issue by amrnt** [brianegan/scoped_model#6]
I see that the original repo makes some changes so the link in the README is not pointing to where the code was extracted from anymore.

And I notice that the `model` has its own library now here: https://github.com/fuchsia-mirror/topaz/tree/master/public/lib/widgets/dart

And I was able to run the example in this repo with this fetching this dep:

```
dependencies:
  lib.widgets.dart:
    git:
      url: https://github.com/fuchsia-mirror/topaz.git
      path: public/lib/widgets/dart # for path from git: documentation is not merged yet: https://github.com/dart-lang/site-www/issues/314
```

as well as this import `import 'package:lib.widgets.dart/model.dart';`

**Comment by brianegan** [comment]
Thanks! I'll fix that up.

**Comment by brianegan** [comment]
Fixed as part of `0.3.0`!
